bridges/snowbridge/primitives/router/src/outbound/mod.rs

```diff
@@ -291,6 +291,7 @@ where
 				match inner_location.unpack() {
 					(0, [AccountKey20 { network, key }]) if self.network_matches(network) =>
 						Some((H160(*key), *amount)),
+					(0, []) => Some((H160([0; 20]), *amount)),
```

Comment:
Add comment for this case explaining why what we are trying to do here, for example, `UnlockNativeToken` command supports unlocking ether if token address is NULL.

Reply:
Address in [d7b33dc](https://github.com/paritytech/polkadot-sdk/pull/6855/commits/d7b33dc8e80776abc8b34fd5fc64f478a05d966c).